### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by 8prsps** [FunkinCrew/Funkin#1969] [Q&A]
Do you know how to add new options to the menu?

**Comment by Snirozu** [comment]
in MainMenuState add new key to the optionShit variable for example: ```['story mode',  'freeplay', 'donate', 'options', 'yourOptionHere'];```

if you want to make it do something go to line 174 and add new case with your option name that you added to optionShit

to make the option have a cool ass image you need to edit FNF_main_menu_assets, 
the fla should be in art/flashFiles
after you open the fla you hate to add two new vectors named "yourOptionName basic" and "yourOptionName white" basic is for when you select the option and white is when you don't have it selected
after you finish editing you have to select every vector that ends with "basic" and "white"
then right click and select "generate sparrow atlas" then select the path where FNF_main_menu_assets.xml is located and click export
